src/main/java/Duke.java

```diff
@@ -1,10 +1,177 @@
+import java.util.ArrayList;
+import java.util.Locale;
+import java.util.Scanner;
+
 public class Duke {
+    private static final Scanner in = new Scanner(System.in);
+    private static final ArrayList<Task> taskStore = new ArrayList<Task>();
```

Comment:
I think final is only used for constants and since you are going to modify taskStore you may want to remove the final in this line.